src/jobs/submitJob.tsx

```diff
@@ -668,9 +714,35 @@ function SubmitJob(
           }
         });
       }
+      handleDuplicateValidation(setDuplicateValidation, listOfFiles);
       setValuesPart(listOfFiles);
     }
   };
+  const handleDuplicateValidation = (
+    setDuplicateValidation: any,
+    listOfFiles: any
```

Comment:
ditto

Reply:
resolved. listOfFiles can be dynamic types ( string, array of object )